src/tools/tool.select.js

```diff
@@ -20,11 +20,15 @@ module.exports = function(paper) {
   paper.selectRect = null;
   paper.selectRectLast = null;
   var selectionRectangleScale = null;
+  var previousRatio = 1;
   var selectionRectangleRotation = null;
   var segment, path, selectChangeOnly;
   paper.imageTraceMode = false;
 
-  // Externalize deseletion
+  // Debug
+  // var debugText = null;
```

Comment:
Debug stuff and commented code must be removed for final merge. Code that ships is not storage for unfinished ideas, GIT remembers everything committed anyways.